src/routes.js

```diff
@@ -123,7 +123,7 @@ export default (
             path="/permissionsmanagement"
             exact
             component={PermissionsManagement}
-            routePermissions={RoutePermissions.permissionsManagement}
+            routePermissions={[RoutePermissions.permissionsManagement,RoutePermissions.userPermissionsrManagement]}
```

Comment:
Hey @Lin1404, is there supposed to be a 'r' between Permissions and Management on `RoutePermissions.userPermissionsrManagement` ?